SoC-2016-Microprojects.md

```diff
@@ -19,6 +19,18 @@ libgit2 list of
 projects](https://github.com/libgit2/libgit2/blob/master/PROJECTS.md)
 rather than the list below.*
 
+Git development is based on sending successive versions of patchs or
```

Comment:
Usually we'd say `patches` in English (I thought initially it was just a typo, but TIL that it's `patchs` in French).